docs/release/release_0_4_8.md

```diff
@@ -0,0 +1,157 @@
+- PR: Add interface language selection to preferences (#2466)
+- Add Hookspec for providing sample data (#2483)
+- Add ability to run file as plugin (#2503)
+- Add `layer.source` attribute to track layer provenance (#2518)
+- Add button to drop into debugger in the traceback viewer. (#2534)
+- PR: Add initial welcome screen on canvas (#2542)
+- Text overlay visual (#2595)
+
+## Improvements
+
+- Add stretch to vertical dock widgets (#2154)
+- Use new selection model on existing `LayerList` (#2441)
+- Add bbox annotator example (#2446)
+- Save last working directory (#2467)
+- add name to dock widget titlebar (#2471)
+- Add Qt`ListModel` and `ListView` for `EventedList` (#2486)
+- New new qt layerlist (#2493)
+- Move plugin sorter (#2501)
+- Add ColorManager to Vectors (#2512)
+- PR: Enhance translation methods (#2517)
+- PR: Add typing to schema_version (#2536)
+- PR: Add initial restart implementation (#2540)
+- Add data setter for `surface` layers (#2544)
+- Extract shortcut into their own object. (#2554)
+- example of using magicgui with thread_worker (#2577)
+- Add example tying slider change to point properties change (#2582)
+- Range of label spinbox is more dtype-aware (#2597)
+- PR: Add generic name to unnamed dockwidgets (#2604)
+- PR: Hide ipy interactive option (#2605)
+- PR: Add option to save state separate from geometry (#2606)
+- QtLargeIntSpinbox for label controls (#2608)
+- Support varying number of dimensions during labels painting (#2609)
+
+## Bug Fixes
+
+- PR: Ensure Preferences dialog can only be opened once (#2457)
+- restore QtNDisplayButton (#2464)
+- fix label properties setter (issue #2477) (#2478)
+- Fix labels data setter (#2496)
+- PR: Fixes localization for colormaps (#2498)
+- Small brackets fix for Perfmon (#2499)
+- Misc documentation fixes (#2502)
+- PR: Add try except on safe load (#2505)
+- Be cautious when checking a module's __package__ attribute (#2516)
+- trigger label colormap generation on seed change to fix shuffle bug, addresses #2523 (#2524)
+- Modified quaternion2euler function to cap the arcsin's argument by +-1 (#2530)
+- single line change to track recoloring function (#2532)
+- PR: Handle Escape on Preferences dialog (#2537)
+- PR: Fix close window handling for non-modals (#2538)
+- PR: Fix trans to use new API (#2539)
+- fix set_call_order with missing plugin (#2543)
+- Update conditional to use new selection property (#2557)
+- Fix visibility toggle (and other key events) in new qt layerlist (#2561)
+- Delay importing plugins during settings registration (#2575)
+- Don't create a dask cache if it doesn't exist (#2590)
+- PR: Update model and actions on menu (#2602)
+- Fix z-index of notifications (hidden by welcome window) (#2611)
+- add missing QSpinBox import in Labels layer controls (#2619)
+
+## API Changes
+
+- Remove toggle theme from drop down menu.  (#2462)
+- Add support for passing `shape_type` through `data` attribute for `Shapes` layers (#2507)
+- Deprecate gui qt (#2533)
```

Comment:
DEFINITELY highlight this. 😊 

Also, this is not an API change yet. I think the API changes section should be reserved for code that used to work that now doesn't. So I *think* we actually don't have any API changes in this release?

Reply:
I had it in the actual "highlights" section, but then removed it as it didn't feel like a "highlight".  It's obviously going to need mentioning in the intro.  Do you want an additional bullet point in that section too?